lib/opensusebasetest.pm

```diff
@@ -5,10 +5,19 @@ use testapi;
 
 # Base class for all openSUSE tests
 
+
+sub clear_console() {
+    # 'ctrl-l' does not get queued up in buffer. If this happens to fast, the
+    # screen would not be cleared
+    script_run "clear";
```

Comment:
in an ideal world, yes, but when the machine is under load, aren't we likely to have a situation where either ctrl-l or clear are pressed, the system doesn't enact it, and therefore the screen doesn't get cleared and so the test fails (this is the problem we witnessed that triggered this chain of thought)

However, I agree the wait_idle is a very expensive pain

And therefore, my proposal is to limit this change to only clear_and_verify_console (see #923 ) where the wait_idle serves a purpose before the assert_screen

While this PR is 'tidy', I remain to be convinced it's worth the cost of doing it everywhere